_includes/breadcrumb.html

```diff
@@ -0,0 +1,11 @@
+<div id="breadcrumb">
+ {% if include.root %}
+   <strong>{{ include.root }}</strong> |   
+ {% endif %}
+ {% if include.branch1 %}
```

Comment:
Great idea! Do you think a loop would be better here so more than two items can be created?

Reply:
Not sure how to create a loop, but I guess if we ever wanted to get more than three levels deep we could just change the include to accept that (or just a "branch3"....)

Reply:
Yes. I think, we should include three levels now. It's better to have too many than not enough ;-).

Reply:
OK I'll add a third (BTW just been reading about loops but I'm not sure how to get the alternating linktext/link thing working with that). 

I'll put them on the page then and add to this branch. Would be good to get that in before translation starts.

Reply:
> ll put them on the page then and add to this branch. Would be good to get that in before translation starts.

Agree. 

Same for the linux installation guide.

Reply:
OK I've done that now I think. Note that it's quite a "conservative" breadcrumb because it's not on every page, just the one's that are sort of "children" of others.

Reply:
Great! I've also edited the Linux install pages (client/server). But I think they will soon need to be re-written because they look quite messy...